pkg/testutils/lint/lint_test.go

```diff
@@ -334,6 +335,85 @@ func TestLint(t *testing.T) {
 		}
 	})
 
+	// TestRaftCopyrightHeaders checks that all the source files in pkg/raft have
+	// the original copyright headers from etcd-io/raft, and the modified files
+	// have Cockroach attribution.
+	t.Run("TestRaftCopyrightHeaders", func(t *testing.T) {
+		t.Parallel()
```

Comment:
Nit: makes sense to me to skip this test if a package is specified:

```
		if pkgSpecified {
			skip.IgnoreLint(t, "PKG specified")
		}
```

Reply:
Done, thanks.